R/prepare_data.R

```diff
@@ -1,3 +1,18 @@
+#' @title Check whether a package is installed
+#' @description A small internal function to check whether a required package
+#' is installed.
+#' This function is primarily intended to be used in mocking when testing for
+#' appropriate error messages from functions that rely on a package without
+#' that package being imported in {cfr}.
+#' The only current use case is [prepare_data.incidence2()].
+#' @param x A package name as a string.
```

Comment:
As this argument is being directly passed to `requireNamespace()` you could use the same argument name (`package`) and then in the documentation use `@inheritParams base::requireNamespace` to save yourself the effort. Minor change either way, and there is nothing wrong with what is there now.

Reply:
Implemented in https://github.com/epiverse-trace/cfr/pull/100/commits/e877c4c72e1ab3b06fef7ff5367e6a273f03fca7